x-pack/plugins/triggers_actions_ui/public/common/get_add_connector_flyout.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { lazy, Suspense } from 'react';
+import { EuiLoadingSpinner, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { ConnectorAddFlyoutProps } from '../application/sections/action_connector_form/connector_add_flyout';
```

Comment:
I think this will cause the lazy loading to break (best to validate using the metrics in the CI message.

It might be that using `import type` will fix that though (please double check before merging this :) ):

```suggestion
import type { ConnectorAddFlyoutProps } from '../application/sections/action_connector_form/connector_add_flyout';
```